packages/plugin-cloudflare/src/install-cloudflared.ts

```diff
@@ -1,34 +1,36 @@
 // Install script for cloudflared, derived from https://github.com/JacobLinCool/node-cloudflared
-import * as path from 'path'
+import {basename, dirname, joinPath} from '@shopify/cli-kit/node/path'
```

Comment:
I don't think this file was under test before. Getting the entire thing under test is surely out of scope for this PR, but this might be a good PR to add a baseline test and maybe capture some of the specific changes you made here. WDYT?

Reply:
Yeah makes sense, I'll add some tests for this 👌